### PR TITLE
Add dynamic backref

### PR DESCRIPTION
**Pull request by nikitagromov**
Pivotal story id [#126323605](https://www.pivotaltracker.com/story/show/126323605)

### CODE REVIEW

File: nefertari_sqla/documents.py [hunk 1]
```diff
@@ -24,10 +25,28 @@
...
+
+    def set_session_factory(self, factory):
+        self.session_factory = factory
+
+    def __call__(self):
+        return self.session_factory()
+
+    def restore_default(self):
+        from pyramid_sqlalchemy import Session
+        self.session_factory = Session
```
Comment (nmenardg) [inline nefertari_sqla/documents.py:47]: If I remember correctly, you implemented a very similar class elsewhere (maybe in Celery tasks?)

Do you think you could use the same class everywhere?

File: nefertari_sqla/fields.py [hunk 2]
```diff
@@ -521,3 +529,7 @@ def Relationship(**kwargs):
     rel_field = relationship(rel_document, **rel_kw)
     rel_field._init_kwargs = _init_kwargs
     return rel_field
+
+
+class UnsupportedFiledValueError(Exception):
+    pass
```
Comment (nmenardg) [inline nefertari_sqla/fields.py]: Could you inherit from ValueError?
Also why "Filed"? Is it a typo?

**Comment by nmenardg** [comment]
Approved

File: nefertari_sqla/documents.py (hunk 1 above)
Comment (nikitagromov) [inline nefertari_sqla/documents.py:47]: This changes was skipped in develop. And I this class wasn't implemented in celery tasks. This class was created for use standalone session factory in celery tasks

**Pull request merged**